.claude/scripts/workflow-state.sh

```
@@ -0,0 +1,908 @@
+    echo "  reconcile <state-file>                   Verify state matches reality"
+    echo "  next-action <state-file>                 Determine next auto-continue action"
+    exit 1
+}
+
+# Initialize a new workflow state file
+# Usage: init <feature-id> [--debug|--refactor]
+cmd_init() {
+    local feature_id="$1"
+    local workflow_type="${2:-feature}"
+    local state_file="$STATE_DIR/${feature_id}.state.json"
+    local now=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)